deployment/HDI-Spark/Notebooks/install-livy-sparkmagic-hn.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+#
+# Update Livy Jars
+#
+
+sudo wget https://ruinliureplhdistorage.blob.core.windows.net/ruinliurepl/livy_jar.zip -O livy_jar.zip
+sudo unzip livy_jar.zip
+
+if [ "$(ls -A /usr/hdp/current/livy2-server/jars)" ]; then
+     sudo rm /usr/hdp/current/livy2-server/jars/*
+fi
+if [ "$(ls -A /usr/hdp/current/livy2-server/repl_2.11-jars)" ]; then
+     sudo rm /usr/hdp/current/livy2-server/repl_2.11-jars/*
+fi
+if [ "$(ls -A /usr/hdp/current/livy2-server/rsc-jars)" ]; then
+     sudo rm /usr/hdp/current/livy2-server/rsc-jars/*
+fi
+
+sudo cp livy_jar/jars/* /usr/hdp/current/livy2-server/jars/
+sudo cp livy_jar/repl_2.11-jars/* /usr/hdp/current/livy2-server/repl_2.11-jars/
+sudo cp livy_jar/rsc-jars/* /usr/hdp/current/livy2-server/rsc-jars/
```

Comment:
I believe this can be done with one `cp` command.

Reply:
Changed